core/src/main/kotlin/org/evomaster/core/output/TestSuiteSplitter.kt

```diff
@@ -113,7 +113,7 @@ object TestSuiteSplitter {
                 if(errs.size <= 1){
                     splitResult.splitOutcome = splitByCode(sol, config)
                     // TODO: BMR - what is the executive summary behaviour for 1 or fewer errors?
-                    splitResult.executiveSummary = sol
+                    splitResult.executiveSummary = sol.convertSolutionToExecutiveSummary()
```

Comment:
not sure about this. if there is at most 1 error, then I think there should be no executive summary. so the logic here would be to leave it empty, and then make sure to not crash when checking its presence

Reply:
Thanks, Andrea for your comment. I kept that one as it is. However, before the executive summary file is created, I added a condition that checks if all test cases belong to the success category. If so, the executive summary is not created.